docs/2.0/docs/patcher/concepts/patches.md

```diff
@@ -1,3 +1,48 @@
 # What is a Patcher Patch?
 
-This page is under construction
+A Patcher Patch is a set of instructions that Patcher can carry out to resolve a breaking change. 
+This instruction sheet is delivered by means of a `yaml` file in a specific format:
+
+```yaml
+name: "<name-of-patch>"
+description: "<description for patch>"
+author: <your-name-here> <<your-email-address-here>>
+
+# Optional dependencies. Terrapatch is a typical one
+dependencies:
+  - name: terrapatch
+    version: "0.1.0"
+
+# Steps necessary to resolve breaking change
+steps:
+  - name: "<name-of-step>"
+    run: <command-to-run>
+  - name: "<name-of-second-step>"
+    run: <second-command-to-run>
+  # etc
+```
+
+## As a Consumer of Patches...
+
+All that's required to ingest patches for breaking IaC changes is to run a `patcher update` (with whatever flags you deem appropriate). 
+
+This command can be run locally, in which case, the steps outlined in the patch(es) would be executed within your codebase, and it would be up to you to apply IoC changes, or open them in a PR against your codebase, whichever is appropriate for your use case. 
+
+Additionally, Gruntwork offers a GitHub Action to automate this process for you.
+Using the GitHub action, PRs will be opened against your codebase on a cadence specified by you, against environments, versions, etc. specified by you. 
+The intention with this GitHub action is to leave you, the repo owner, in full control of your upgrade cadence. 
+
+Regardless of methodology, the concept remains the same. 
+Patcher will suggest changes to your codebase in order to keep your infrastructure up to date, however you see fit. 
+
+## As an Author of Patches...
+
+As an author of a patch, this means that you are publishing a breaking change with a release of your module. 
```

Comment:
Switch to more active language. Instead of "this means that you are...," consider language like "Module authors periodically need to introduce breaking changes in their modules, causing a downstream painful experience for module consumers. With patches, module authors write not only the updated module code, but also include a patch YAML file that automatically updates consuming code to incorporate the breaking changes.

use patches to enable their modules consumers to automatically update consuming code to adopt breaking change.

Reply:
i like your verbage! incorporated 👍 

as a general note, you'll likely notice i abstain from assertive/aggressive language. if i swung too far away from the docs site's conventions, please do feel free to call me out, always happy to fix it!